src/Lykke.Service.Regulation/Startup.cs

```diff
@@ -17,6 +17,7 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Swashbuckle.AspNetCore.Swagger;
```

Comment:
please add changes related to ENV_INFO (https://github.com/LykkeCity/lykke.dotnettemplates/commit/bc36f013dc7ab6dc44cd5e772de61cf0ef1de272)

Reply:
done